frontend/src/components/SamplesList.tsx

```diff
@@ -271,6 +291,7 @@ export default function SamplesList({
               <Col className={"text-start"}>
                 <Button
                   className={"btn btn-success"}
+                  disabled={alertContent === costCenterAlertContent}
```

Comment:
Disabled/greyed out the "Submit Updates" button when there is an invalid Cost Center input.